### ISSUE TITLE
Zobrazení čísel podle jiného než juliánského kalendáře

### ISSUE THREAD

**Issue by zabak** [ceskaexpedice/kramerius-web-client#557]
Pokud bude vyřešeno https://github.com/ceskaexpedice/kramerius/issues/907 tak bude potřeba vyřešit ještě zobrazení data vydání pro uživatele (aby pochopil, proč vidí jiné datum vydání v kalendáři a vytištěné v novinách). Např. zobrazit v metadatech mimo data podle gregoriánského kalendáře do závorky ještě datum podle juliánského).

**Comment by honza-rychtar** [comment]
@zabak bude se tohle realizovat? Bylo be nejdřív potřeba opravdu tu informaci do dat zapisovat a taky upravit indexer.

**Comment by zabak** [comment]
Ano, ale ještě to není ve standardu ani v digitalizačních linkách. Do budoucna to bude důležité pro hledání v novinách podle dní.
